studio/schemas/documents/benefit.ts

```diff
@@ -1,27 +1,52 @@
-import { defineType } from 'sanity';
-import { richText, title } from '../fields/text';
+import { defineField, defineType } from "sanity";
+import { richText, title } from "../fields/text";
 
 export const benefitId = "benefit";
+export const benefitTypeId = "benefitType";
+
+const BENEFIT_TYPE_BASIC_VALUE = "basic";
+const BENEFIT_TYPES = [
+  { title: "Basic", value: BENEFIT_TYPE_BASIC_VALUE },
+  { title: "Gadgets", value: "gadgets" },
+  { title: "Bonus", value: "bonus" },
+  { title: "Pension", value: "pension" },
+  { title: "Salary Growth", value: "salaryGrowth" },
+];
+
+const benefitType = defineField({
+  name: benefitTypeId,
+  type: "string",
+  title: "Benefit Type",
+  description:
+    "Choose the type of benefit. Some benefit types include visual graphs that will be displayed together with the text.",
+  options: {
+    list: BENEFIT_TYPES,
+    layout: BENEFIT_TYPES.length > 5 ? "dropdown" : "radio",
+  },
+  initialValue: BENEFIT_TYPE_BASIC_VALUE,
```

Comment:
is it correct that initial value can be a string while the options are object values?

Reply:
Yes, `initialValue` must be a `string` (or a function resolving to a `string`)